src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -680,6 +686,12 @@ gl_sarray drawing_classifier::predict(gl_sframe data, std::string output_type) {
                   "Expected one of: probability, probability_vector, rank");
   }
 
+  std::string feature_column_name = read_state<flex_string>("feature_column_name");
+  if (!(data.contains_column(feature_column_name))) {
```

Comment:
consider using ASSERT_MSG for a sanity check.

Reply:
Why should we have an assert message instead of throwing an error with a message to the user?